src/browser_action/vitals.js

```diff
@@ -234,7 +240,7 @@
           <div class="lh-metric__innerwrap">
             <div>
               <span class="lh-metric__title">
-                Largest Contentful Paint 
+                Largest Contentful Paint${' '}
```

Comment:
just a fix to prevent editors from removing this whitespace since it seems important

Reply:
👍